command/src/main/java/pw/chew/jdachewtils/command/OptionHelper.java

```diff
@@ -0,0 +1,171 @@
+package pw.chew.jdachewtils.command;
+
+import net.dv8tion.jda.api.entities.GuildChannel;
+import net.dv8tion.jda.api.entities.IMentionable;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * A collection of useful methods for working with Options.
+ */
+public final class OptionHelper {
+    private OptionHelper() {}
+
+    /**
+     * Guarantees a String option value by providing a default value.
+     *
+     * @param event        the slash command event to get options from
+     * @param option       the option we want
+     * @param defaultValue if the option doesn't exist, what should we use instead?
+     * @return the never-null option
+     */
+    public static String optString(@NotNull SlashCommandEvent event, String option, String defaultValue) {
```

Comment:
```java
@Nullable
@Contract("_, _, !null -> !null")
```
Add these annotations for the methods which does not return primitive types like String, GuildChannel, Member, IMentionable, Role, User, MessageChannel. This will help developers to understand what the behavior of the method when you give null object for the default value. And also you will be able to see if it will return null or not on IntellijIDEA.

Reply:
This is pretty cool, added and update annotations for all the parameters, and updated the documentation to reflect more the return type more accurately